opm/simulators/timestepping/AdaptiveTimeSteppingEbos.hpp

```diff
@@ -361,6 +354,7 @@ namespace Opm {
 
                     ++restarts;
                 }
+                ebosProblem.setNextTimeStepSize(substepTimer.currentStepLength());
```

Comment:
The next time step is now quarried from the simulator instead of passed to the output method. This line communicates the next time step to the simulator.